CovertInfiltration/Localization/CovertInfiltration.int

```diff
@@ -107,6 +104,9 @@ strBannerBonusGained="Overinfiltration bonus gained"
 BonusName="TacticalAnalysis"
 BonusDescription="TacticalAnalysis TacticalAnalysis TacticalAnalysis"
 
-[Activity_NeutralizeCommander X2ActivityTemplate_Infiltration]
-ActionRewardDisplayName=Rwd
-ActionRewardDetails=Rwd details
\ No newline at end of file
+;[CovertAction_CommanderSupplyInfil X2CovertInfiltrationTemplate]
+;ActionObjective="Neutralize Field Commander"
+
+;[Activity_CommanderSupply X2ActivityTemplate_Infiltration]
+;ActionRewardDisplayName=Rwd
+;ActionRewardDetails=Rwd details
```

Comment:
What's the purpose of leaving this here?

Reply:
To remind myself that I need to make those localization entries for every activity